### PR TITLE
Reverting to regularizing logits in LS/TR term

### PR DESCRIPTION
**Pull request by ajratner**
Works significantly better, tested on CIFAR-10 to confirm; hardcoded conservative defaults (ls_term=0.1, ls_term_n_passes=1, batch_size_u=10, batch_size=50) based on best-scoring CIFAR-10 model with LS/TR term

### CODE REVIEW

**Comment by henryre** [comment]
Looks good. Was there another change re: batch mean that needed to happen?

**Comment by henryre** [comment]
^ there wasn't
